infrastructure/main.tf

```diff
@@ -80,6 +80,12 @@ module "db" {
     {
       name  = "rds.force_ssl"
       value = 1
+    },
+    # Set timeout for idle connections to 24 hours in milliseconds
+    # See https://postgresqlco.nf/doc/en/param/idle_session_timeout/
+    {
+      name = "idle_session_timeout"
```

Comment:
Is this supported with postgres version 13? 😊  I can't find it in the docs.

Reply:
Good point. Should we upgrade to postgresql 14 then? That is the stable version

Reply:
nyt nyt nyt!

Reply:
Upgraded to postgres version 14